packages/@ember/deprecated-features/index.ts

```diff
@@ -13,3 +13,4 @@ export const FUNCTION_PROTOTYPE_EXTENSIONS = !!'3.11.0-beta.1';
 export const MOUSE_ENTER_LEAVE_MOVE_EVENTS = !!'3.13.0-beta.1';
 export const PARTIALS = !!'3.15.0-beta.1';
 export const GLOBALS_RESOLVER = !!'3.16.0-beta.1';
+export const ASSIGN = !!'4.0.0';
```

Comment:
```suggestion
export const ASSIGN = !!'4.0.0-beta.1';
```

This will be the first version with the deprecation